src/main/release-scripts/upload-distribution.sh

```diff
@@ -1,11 +1,27 @@
 #!/usr/bin/env bash
 
-DIST_PARENT_DIR=$1
-RELEASE_VERSION=$2
+if [ -z "$1" ]
+  then
+    echo "Release version not supplied"
+    exit 1
+fi
```

Comment:
Yes, but I'll move the check on RELEASE_VERSION instead of 1 to make it clearer what I'm validating